packages/shopware-6-client/src/services/checkoutService.ts

```diff
@@ -0,0 +1,32 @@
+import { apiService } from "../apiService";
+import {
+  getCheckoutOrderEndpoint,
+  getCheckoutGuestOrderEndpoint
+} from "../endpoints";
+import { Order } from "@shopware-pwa/shopware-6-client/src/interfaces/models/checkout/order/Order";
+
+/**
```

Comment:
please provide docs with explaining when to use this and when guestOrder

Reply:
done :+1: